.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: ruby
 rvm:
-  - 3.1.3
+  - 3.4.5
```

Comment:
I'm pretty positive this whole file can go because I don't think we use travis ci anywhere